### PR TITLE
feat(select-status): the style modification due to the 'ic_check' icon change & minor bug

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [x] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [ ] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
- the style modification due to the 'ic_check' icon change
image

- fixed storybook path bug at toolbox-table

### CODE REVIEW

File: packages/mirinae/src/data-display/tables/toolbox-table/PToolboxTable.stories.mdx [hunk 1]
```diff
@@ -7,6 +7,7 @@ import PSelectStatus from '@/inputs/select-status/PSelectStatus';
 import PToolboxTable from './PToolboxTable.vue';
 import { getToolboxTableArgTypes } from '@/data-display/tables/toolbox-table/story-helper';
 import { getUserFields, getUsers } from '@/data-display/tables/data-table/mock';
+import { useProxyValue } from "@/hooks/proxy-state/index";
```
Comment (yuda110) [inline packages/mirinae/src/data-display/tables/toolbox-table/PToolboxTable.stories.mdx:10]: praise: 🦅

Comment (WANZARGEN) [inline packages/mirinae/src/data-display/tables/toolbox-table/PToolboxTable.stories.mdx:10]: suggestion: 😄

```suggestion
import { useProxyValue } from "@/hooks/proxy-state";
```

**Review by WANZARGEN — APPROVED** [review]
Goood

**Review by yuda110 — APPROVED** [review]
LGTM!

**Pull request merged**